src/ingest-contact-loaders/csv-upload/index.js

```diff
@@ -0,0 +1,136 @@
+import { completeContactLoad } from "../../workers/jobs";
+import { r, CampaignContact } from "../../server/models";
+import { getConfig, hasConfig } from "../../server/api/lib/config";
+
+import { updateJob } from "../../lib";
+import { getTimezoneByZip, unzipPayload } from "../../workers/jobs";
+
+export const name = "csv-upload";
+
+export function displayName() {
+  return "CSV Upload";
+}
+
+export function serverAdministratorInstructions() {
+  return {
+    "environmentVariables": [],
+    "description": "CSV Upload a list of contacts",
+    "setupInstructions": "Nothing is necessary to setup since this is default functionality"
+  }
+}
+
+export async function available(organization, user) {
+  /// return an object with two keys: result: true/false
+  /// if the ingest-contact-loader is usable and has
```

Comment:
```suggestion
  /// these keys indicate if the ingest-contact-loader is usable
```

I think this is the sentence you meant? This is a nit, it's just not a fully formed comment